src/lib/skilling/skills/hunter/creatures/bso.ts

```diff
@@ -34,6 +34,25 @@ const customBSOCreatures: Creature[] = [
 			req.add('Simple kibble', kibbleRequired);
 			return req;
 		}
+	},
+	{
+		name: 'Eastern ferret',
+		id: 91_938,
+		aliases: ['eastern ferret'],
+		level: 96,
+		hunterXP: 2100,
+		table: new LootTable().every('Eastern ferret'),
+		qpRequired: 600,
+		huntTechnique: HunterTechniqueEnum.Tracking,
+		catchTime: 350,
+		slope: 0,
+		intercept: 99,
+		bait: qty => {
+			let req = new Bank();
+			const kibbleRequired = Math.ceil(qty / 2);
+			req.add('Extraordinary kibble', kibbleRequired);
```

Comment:
There's no current use for 'Delicious kibble' yet which is the middle tier, perhaps this should use 'Delicious kibble', and save 'Extraordinary kibble' for a higher level thing?